src/migration.js

```diff
@@ -54,24 +54,24 @@ module.exports = class Migration {
       })();
     }
 
-    return require(this.path);
+    return import(this.path);
```

Comment:
If you aren't awaiting anything, don't mark the method as async. (Unless import actually returns Object and you are depending on the compiler to wrap it in a Promise.resolve())